src/main/java/com/github/maxopoly/kira/api/sessions/APIIngameCommandSession.java

```diff
@@ -0,0 +1,24 @@
+package com.github.maxopoly.kira.api.sessions;
+
+import org.json.JSONObject;
+
+import com.github.maxopoly.kira.api.input.APISupplier;
+import com.github.maxopoly.kira.rabbit.session.SendIngameCommandSession;
+
+public class APIIngameCommandSession extends SendIngameCommandSession {
+	
+	private APISupplier supplier;
+	
+	public APIIngameCommandSession(APISupplier supplier, String command) {
+		super(supplier, command);
+	}
+
+	@Override
+	public void handleReply(JSONObject json) {
+		String reply = json.getString(replyField);
+		if (reply.trim().length() == 0) {
+			reply = "Server sent empty reply";
```

Comment:
No reply should be different from the actual reply "Server sent empty reply". If no reply is given, the API should be given an empty string or no reply at all (null in the method call and then not included as option in the reply json)